src/mem/ghcb.rs

```diff
@@ -10,7 +10,7 @@ use crate::cpu::percpu::PERCPU;
 use crate::cpu::percpu_count;
 use crate::cpu::vc_register_ghcb;
 use crate::cpu::vc_terminate;
-use crate::dynam::mem_allocate_frames;
+use crate::mem::mem_allocate_frames;
```

Comment:
This breaks rustfmt, rearrange following rustfmt --check. You can also just do rustfmt src/lib.rs

Reply:
Done